pkg/coreapi/gql.schema.graphql

```diff
@@ -550,3 +551,12 @@ type ConnectV1WorkerConnectionEdge {
   node: ConnectV1WorkerConnection!
   cursor: String!
 }
+
+enum AppConnectionType {
+	SERVERLESS
+	CONNECT
```

Comment:
According to the designs the types would be serverless and persistent. Did the term change?

Reply:
Going by Dan's comment here, distinguishing not on the app type but the connection type (serverless vs. worker/connect). Persistent feels somewhat confusing